src/plugins/notebook/utils/notebook-storage.js

```diff
@@ -48,6 +54,24 @@ export function getDefaultNotebook() {
     return JSON.parse(notebookStorage);
 }
 
+export function getNotebookSectionAndPage(domainObject, sectionId, pageId) {
+    const section = domainObject.configuration.sections.find(s => s.id === sectionId);
+    const page = section && section.pages.find(p => p.id === pageId);
+
+    if (!section || !page) {
+        console.warn('Clearing default notebook due to missing Section or Page from default Notebook');
+        console.warn('Section:', section);
+        console.warn('Page:', page);
+
+        clearDefaultNotebook();
```

Comment:
Since this is a `get` method it's odd that its changing the notebook storage by clearing it as a side effect.
Can we separate the clearing of the notebook from getting the section and page?